<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInfo</class>
 <widget class="QWidget" name="UserInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: none;</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QWidget" name="BottomWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
    <widget class="QPushButton" name="LogOut">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-1</y>
       <width>300</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    font-size: 23px;
	border: none;
     background-color: rgb(255, 255, 255);
    }
QPushButton:hover {
    background-color: rgb(242, 243, 244);
    }</string>
     </property>
     <property name="text">
      <string>退出登录</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="CenterWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
</string>
    </property>
    <widget class="QPushButton" name="Personal_Information_Settings">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>300</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    font-size: 23px;
	border: none;
     background-color: rgb(255, 255, 255);
    }
QPushButton:hover {
    background-color: rgb(242, 243, 244);
    }</string>
     </property>
     <property name="text">
      <string>个人信息设置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="MyMembers">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>300</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    font-size: 23px;
	border: none;
     background-color: rgb(255, 255, 255);
    }
QPushButton:hover {
    background-color: rgb(242, 243, 244);
    }</string>
     </property>
     <property name="text">
      <string>我的会员：当前无会员</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="TopWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>161</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 25px;</string>
     </property>
     <property name="text">
      <string>璐璐家投了</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Profile_Picture">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>15</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid rgb(226, 229, 233);
border-radius: 30px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>199</y>
      <width>260</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:  none;
background-color: rgba(184, 189, 198, 0.5);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>99</y>
      <width>260</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:  none;
background-color: rgba(184, 189, 198, 0.5);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
